### DISCUSSION TITLE
Pgmoneta-admin master-key can't use letters

### DISCUSSION THREAD

**Discussion by bokket** [pgmoneta/pgmoneta#114] [General]
It looks like this:
![深度截图_选择区域_20230315145146](https://user-images.githubusercontent.com/57066972/225229395-137d2337-58fb-429f-830f-f2bd4dc5a2b5.png)
I think it's like a bug that you can't enter letters, it always skips lines without showing any information until you enter a number, we should pass in a string parameter.

**Comment by jesperpedersen** [comment]
Are you using `A`-`Z` + `a`-`z` letters ?

**Reply by bokket** [reply]
yes.Is it not allowed?

**Reply by jesperpedersen** [reply]
Ok, it is working here.

Can you `gdb` it on your side ?

**Reply by bokket** [reply]
I use to`b 100` point to find this link [MAX_PASSWORD_LENGTH=1024](https://github.com/pgmoneta/pgmoneta/blob/main/src/include/pgmoneta.h#L60).
Ref [logic](https://github.com/pgmoneta/pgmoneta/blob/main/src/libpgmoneta/utils.c#L535).I try again to find that must enter>=8 characters(It does not matter if it is a letter or not).why?

**Reply by jesperpedersen** [reply]
No, it shouldn't matter
